packages/frontend-shared/src/locales/en-US.json

```diff
@@ -426,7 +426,7 @@
       "errorNotFound": "Project not found",
       "errorNotFoundButton": "Connect it again",
       "errorNotLoggedIn": "You are not connected to the Cypress Dashboard",
-      "errorNotLoggedInButton": "Log in to Cypress Dashboard"
+      "errorNotLoggedInButton": "Log in to the Cypress Dashboard"
```

Comment:
I'm trying to figure out where this is used - can't find usages of `errorNotLoggedInButton`.

Reply:
Looks like several of these strings aren't referenced (most of the `error*` entries under `recordKey` anyways)